wgsl/index.bs

```diff
@@ -3583,7 +3583,7 @@ but potentially with a different representation.
 It can be read using [[#textureload|textureLoad]] or [[#texturesamplelevel|textureSampleLevel]] built-in functions,
 which handle these different representations opaquely.
 
-See [[WebGPU#GPUExternalTexture]].
+See [[WebGPU#gpu-external-texture|WebGPU &sect; GPUExternalTexture]].
```

Comment:
This makes the nice-looking "section" sign in the middle.


![Screen Shot 2022-09-13 at 4 07 50 PM](https://user-images.githubusercontent.com/13054574/189999203-f76c6145-1d95-42d3-adba-72c28060af95.png)

Elsewhere in the spec it's made automatically when referencing GPUProgrammableStage because of a happy accident about how the API spec names its section.  But here I made it manually with the `&sect;` element.